src/controllers/users.js

```diff
@@ -0,0 +1,23 @@
+import bcrypt from "bcrypt"
+import utils from "../utils/index"
+import db from "../database/models/index"
+const User = db.User
+let newUser;
+
+exports.createUser = async(req, res) => {
+    try {
+        const salt = await bcrypt.genSalt(10)
+        hashedPassword = await bcrypt.hash(req.body.password, salt)
+        newUser = await User.create({
+            name: req.body.name,
+            email: req.body.email,
+            password: hashedPassword
+        })
+        const tokenObject = utils.issueJWT(user);
+        res.set("Authorization", tokenObject.token)
+        res.status(200).json({ success: true, token: tokenObject.token, expiresIn: tokenObject.expires });
+    } catch (err) {
+        res.status(400).send({ error: err })
```

Comment:
Catch throws Internal server error not bad request. Learn about HTTP status code